x/mongo/driver/auth/oidc.go

```diff
@@ -13,29 +13,27 @@ import (
 	"io"
 	"net/http"
```

Comment:
Again we are seeing some changes from the end of the human flow PR here. The relevant changes are just exporting some of these values to make use of them in configuration errors.